src/main/java/com/nutomic/syncthingandroid/syncthing/DeviceStateHolder.java

```diff
@@ -54,8 +59,13 @@ public boolean isWifiConnected() {
         return mIsWifiConnected;
     }
 
+    public boolean isAPOn() {
```

Comment:
This should be `isApOn` for better readability (I like `isApEnabled` more). Same for other occurences.